crt_portal/cts_forms/forms.py

```diff
@@ -1263,35 +1263,36 @@ class PrintActions(Form):
 
 
 class BulkActions(Form, ActivityStreamUpdater):
+
     assigned_section = ChoiceField(
         label='Section',
         widget=ComplaintSelect(
             attrs={'class': 'usa-select text-bold text-uppercase crt-dropdown__data'},
         ),
-        choices=_add_empty_choice(SECTION_CHOICES),
+        choices=_add_empty_choice(SECTION_CHOICES, default_string='Multiple'),
```

Comment:
Small opportunity here to define `Multiple` in a single place and not hard code it across all of our fields here.